pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools>=40.8.0", "ninja", "wheel"]
+requires = ["setuptools>=40.8.0", "ninja", "wheel", "pybind11==2.6.0rc3"]
```

Comment:
why `==` rather than `>=` for the requirement here?

Reply:
pyproject.toml's usually pin requirements fairly tightly; it can be used to create a completely reproducible build. Since the items here are _always_ downloaded and installed into the temporary build environment, there's no need to make the requirements too wide. Though it could be loosened up if you want. I think 2.6.0 will be out Monday-ish; then you can change this. I would probably not do more than `"pybind11~=2.6"`, though. (The tilde means the final digit, the 6 in this case, can be that or higher)

Reply:
My recommendation would be to keep it pinned to match the submodule, though.

I realize that I forgot to drop this from the MANIFEST, will do that.

Reply:
Here's an example of when it is very important to pin exactly: https://scikit-hep.org/developer/packaging#special-additions-numpy

Reply:
thanks, makes sense!